package.json

```diff
@@ -10,9 +10,10 @@
     "preact"
   ],
   "scripts": {
-    "all": "run-s lint build test",
-    "build": "del-cli preact dist && rollup -c",
```

Comment:
So we don't clean up preact anymore?

Reply:
My bad, fixed via https://github.com/bpmn-io/properties-panel/pull/304/commits/961825cf25d7b7e10f6e40d5e3c9e1b78d3a07f2.